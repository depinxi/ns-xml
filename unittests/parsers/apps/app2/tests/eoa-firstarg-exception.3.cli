-b "Argument" --multi-argument=- "Ignore eoa if it's the first argument"