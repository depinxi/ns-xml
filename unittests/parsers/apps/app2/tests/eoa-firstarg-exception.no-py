CLI: "-b", "Argument", "-c", "-", "Ignore eoa if it's the first argument"
Value count: 1
Values: "Ignore eoa if it's the first argument"
Error count: 0
Subcommand: 
switchA=False
argumentB=Argument
multiargC=
