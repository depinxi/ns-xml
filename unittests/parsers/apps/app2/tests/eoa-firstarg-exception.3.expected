CLI: "-b", "Argument", "--multi-argument=-", "Ignore eoa if it's the first argument"
Value count: 0
Values: 
Error count: 0
Subcommand: 
switchA=False
argumentB=Argument
multiargC=- Ignore eoa if it's the first argument
